pkg/conf/schedulerconf.go

```diff
@@ -68,12 +70,12 @@ type SchedulerConf struct {
 }
 
 func GetSchedulerConf() *SchedulerConf {
-	return configuration
+	return (*SchedulerConf)(atomic.LoadPointer(&pointer))
 }
 
 // unit tests may need to override configuration
```

Comment:
The comment should be modified, because not only the tests use this function anymore.